Lombiq.ContentEditors.Samples/Migrations/EmployeeMigrations.cs

```diff
@@ -9,6 +9,7 @@
 
 namespace Lombiq.ContentEditors.Samples.Migrations;
 
+// It's the migration class for the Employee content type. Nothing specific to async editors here.
```

Comment:
Change all of these "it's" to "this is".

Reply:
**All** of these. ContentEditorsSamplesNavigationProvider and SupportTicketAsyncEditorProvider still.